### ISSUE TITLE
Exception thrown when viewing test results

### ISSUE THREAD

**Issue by rjmartell** [opengeospatial/ets-wms-client13#17]
An exception report is displayed after the test run is finished and the results are presented. From the Tomcat stderr log:

```
Error on line 21 of /teamengine/WEB-INF/classes/com/occamlab/te/logstyles/result-log.xsl:
  FODC0002: Invalid URI {C:\srv\teamengine\users\ogctes...} passed to collection() function
Jan 29, 2016 12:51:17 PM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet [jsp] in context with path [/teamengine] threw exception 
[An exception occurred processing JSP page /viewSessionLog.jsp at line 110

109:       ArrayList tests = new ArrayList();
110:       boolean complete = ViewLog.view_log(suiteName,userlog, sessionId, tests, ViewLogTemplates, out);     
111:       out.println("<br/>");
112:       if (!complete) {

Stacktrace:] with root cause; 
SystemID: file:/C:/srv/tomcat/base-2/webapps/teamengine/WEB-INF/classes/com/occamlab/te/logstyles/result-log.xsl; Line#: 21; Column#: -1
net.sf.saxon.trans.XPathException: Invalid URI {C:\srv\teamengine\users\ogctes...} passed to collection() function
```

The stylesheet 'result-log.xsl' is part of the `teamengine-resources` module, but it appears to be used exclusively to assess the coverage achieved by a WMS client.

**Note**: Applies to ets-wms-client13-1.2-SNAPSHOT

**Comment by rjmartell** [comment]
This is line 21 in result-log.xsl:

```
<xsl:for-each select="collection(concat($logdir,'/',$sessionDir,'?select=WMS-*.xml'))">
```

The $logdir parameter is an absolute file system path, but the [collection() function](https://www.w3.org/TR/xpath-functions/#func-collection) expects a URI.

**Comment by keshavnangare** [comment]
Fixed with this commit [c127fce6716089bdbfc7cbf502ad0bc84e24fe32](https://github.com/opengeospatial/teamengine/commit/c127fce6716089bdbfc7cbf502ad0bc84e24fe32)
